pkg/kubelet/kuberuntime/kuberuntime_image.go

```diff
@@ -90,12 +90,7 @@ func (m *kubeGenericRuntimeManager) GetImageRef(image kubecontainer.ImageSpec) (
 	if status == nil {
 		return "", nil
 	}
-
-	imageRef := status.Id
-	if len(status.RepoDigests) > 0 {
-		imageRef = status.RepoDigests[0]
-	}
-	return imageRef, nil
+	return status.Id, nil
```

Comment:
do you know if there are any unintended consequences of this change to always return ID?

Reply:
it looks like the only real caller is in the image manager itself, so it doesnt look like there are side-effects.